### PR TITLE
Documentation for hook_civicrm_alterEntityRefParams

### PR DESCRIPTION
**Pull request by monishdeb**

### CODE REVIEW

**Comment by monishdeb** [comment]
@seanmadsen can you please review my patch? Do I need to change anything?

**Comment by seanmadsen** [comment]
Thanks @monishdeb :grin: I added some minor adjustments. Ready to merge once the core PR is merged.

One thing to note here (you don't need to remember this &mdash; I'm just pointing it out because I think it's interesting)... that hooks summary page is auto-generated by running `./bin/hook-summary.py`. I adjusted the hook page to use the more detailed summary you wrote, and then re-generated the summary page. The summary page re-gen seems to have picked up some edits that someone made to the menu to improve consistency of capitalization. (That's why the diff on the summary page shows so many lines changed.)

**Comment by monishdeb** [comment]
Ahh nice .. thanks @seanmadsen

File: docs/hooks/hook_civicrm_alterEntityRefParams.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Parameters
+
+- array `$params` - parameters of entityRef field
+- string `$formName` - form name
+
+## Example
+
+```php
+function myextension_civicrm_alterEntityRefParams(&$params) {
```
Comment (seamuslee001) [inline docs/hooks/hook_civicrm_alterEntityRefParams.md]: @monishdeb i think this needs to have 

```php
, $formName
``` Also you need a $ in L25 for form name

Comment (monishdeb) [reply docs/hooks/hook_civicrm_alterEntityRefParams.md]: Oops... thanks @seamuslee001 for pointing that out

**Comment by monishdeb** [comment]
@seanmadsen the core PR https://github.com/civicrm/civicrm-core/pull/11192 is merged now. Can you please merge this PR?

**Comment by seamuslee001** [comment]
Documentation of hook looks good, core PR has merged so merging

**Pull request merged**